pysparkling/fileio/fs/s3.py

```diff
@@ -1,8 +1,8 @@
 from __future__ import absolute_import
 
-import fnmatch
+from fnmatch import fnmatch
 import logging
-from io import BytesIO, StringIO
```

Comment:
Oh yes I missed that during the rebase. I will update the PR tomorrow,
unless you want to merge as is and fix this.

Le mar. 26 janv. 2016 19:53, Sven Kreiss <notifications@github.com> a
écrit :

> In pysparkling/fileio/fs/s3.py
> <https://github.com/svenkreiss/pysparkling/pull/20#discussion_r50880843>:
>
> >  import logging
> > -from io import BytesIO, StringIO
>
> That's what just changed this morning. It's used now in load_text().
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/svenkreiss/pysparkling/pull/20/files#r50880843>.
>